ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -879,7 +879,7 @@ mca_pml_ucx_send_nb(ucp_ep_h ep, const void *buf, size_t count,
         return OMPI_SUCCESS;
     } else if (!UCS_PTR_IS_ERR(req)) {
         PML_UCX_VERBOSE(8, "got request %p", (void*)req);
-        MCA_COMMON_UCX_WAIT_LOOP(req, ompi_pml_ucx.ucp_worker, "ucx send", ompi_request_free(&req));
+        MCA_COMMON_UCX_WAIT_LOOP(req, ompi_pml_ucx.ucp_worker, "ucx send", ucp_request_free(req));
```

Comment:
do we need to reset the request here by `mca_pml_ucx_request_reset`, e.g, call `mca_pml_ucx_request_free` instead (or similar one with proper semantic)?

Reply:
actually seems like we don't modify the ompi_request_t part of UCP request in this function, so ucp_request_free would be enough

Reply:
`ucp_request_free` should enough